server/src/main/java/org/cloudfoundry/identity/uaa/db/SetDefaultSamlSignatureAlgorithm_4_10_0.java

```diff
@@ -0,0 +1,86 @@
+package org.cloudfoundry.identity.uaa.db;
+
+import org.cloudfoundry.identity.uaa.util.JsonUtils;
+import org.cloudfoundry.identity.uaa.zone.IdentityZone;
+import org.cloudfoundry.identity.uaa.zone.IdentityZoneConfiguration;
+import org.cloudfoundry.identity.uaa.zone.SamlConfig;
+import org.flywaydb.core.api.migration.spring.SpringJdbcMigration;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.PreparedStatementSetter;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.util.StringUtils;
+
+import java.util.Date;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.List;
+
+import static org.cloudfoundry.identity.uaa.zone.SamlConfig.SignatureAlgorithm.SHA1;
+
+public class SetDefaultSamlSignatureAlgorithm_4_10_0 implements SpringJdbcMigration {
```

Comment:
Did we verify that we needed this migration? I thought we were going to test with GESSO with sha256 and if it worked we would just put out the breaking change?

Reply:
GE SSO works with `SHA256`. Migration scripts can be removed